elastic/kibana
# [Infrastructure UI] Fix: Expandable content inside the inventory details page

Closes [#153373](https://github.com/elastic/kibana/issues/153373)

## Summary

This PR fixes the issue of showing the `+1 more` link when there is only one value.

## Testing 
- Open the inventory page and select one of the hosts
- Select Metadata inside the details modal window
   - If there is only one value the  `+1 more`  should not be visible
   - If there are more than 1 values the `+n more` (n is the number of values after the first one) should be available and functional: 

   https://user-images.githubusercontent.com/14139027/226663607-bd01ab67-75d0-45e0-9244-0818aff3c0e4.mov
